package.json

```diff
@@ -87,19 +87,22 @@
     "vscode:prepublish": "yarn run compile",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
-    "test": "yarn run compile && node ./node_modules/vscode/bin/test"
+    "test": "node ./out/test/runTests"
```

Comment:
I tried running this:
```
yarn
yarn test
```
But I got this error:

```
$ yarn test
yarn run v1.22.17
warning vscode-fish@1.0.27: The engine "vscode" appears to be invalid.
$ node ./out/test/runTests
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/brian.malehorn/vscode-fish/out/test/runTests'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Could you change this line of code? I believe this will fix the issue:
```json
    "test": "yarn run compile && node ./out/test/runTests"
```

Reply:
Thanks, I'll change this asap!